cmake/configs/posix_sitl_default.cmake

```diff
@@ -33,7 +33,7 @@ set(config_module_list
 	modules/sensors
 	modules/simulator
 	modules/mavlink
-	modules/attitude_estimator_ekf
+	#modules/attitude_estimator_ekf
```

Comment:
We either should remove the code or not comment it. We need to build stuff that is in the codebase to ensure it doesn't bit-rot.